src/router/index.ts

```diff
@@ -4,7 +4,7 @@ import Router from 'vue-router';
 import Login from '@/containers/Login.vue';
 import Dashboard from '@/containers/Dashboard.vue';
 import Page404 from '@/containers/404.vue';
-import store from '@/store';
+import store from '../store';
```

Comment:
Is it better to use `../` here instead of `@/`? I thought `@` referred to `src`.

Reply:
It is used to make the ts project for requirement generator happy